source/benchmarking/benchmarking.md

```diff
@@ -0,0 +1,689 @@
+| -------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Composite name | Compositing formula                                                                                                                                                                                     |
+| BSONBench      | Average of all BSON micro-benchmarks                                                                                                                                                                    |
+| SingleBench    | Average of all Single-doc micro-benchmarks, except "Run Command"                                                                                                                                        |
+| MultiBench     | Average of all Multi-doc micro-benchmarks                                                                                                                                                               |
+| ParallelBench  | Average of all Parallel micro-benchmarks                                                                                                                                                                |
+| ReadBench      | Average of "Find one", "Find many and empty cursor", "GridFS download", "LDJSON multi-file export", and "GridFS multi-file download" microbenchmarks                                                    |
+| WriteBench     | Average of "Small doc insertOne", "Large doc insertOne", "Small doc bulk insert", "Large doc bulk insert", "GridFS upload", "LDJSON multi-file import", and "GridFS multi-file upload" micro-benchmarks |
+| DriverBench    | Average of ReadBench and WriteBench                                                                                                                                                                     |
+
+At least for this first DriverBench version, scores are combined with simple averages. In addition, the BSONBench scores
+do not factor into the overall DriverBench scores, as encoding and decoding are inherent in all other tasks.
+
+## Benchmark platform, configuration and environments
+
+### Benchmark Client
+
+TBD: spec Amazon instance size; describe in general terms how language clients will be run independently; same AWS zone
+as server
+
+All operations must be run with write concern "w:1".
+
+### Benchmark Server
+
+TBD: spec Amazon instance size; describe configuration (e.g. no auth, journal, pre-alloc sizes?, WT with compression to
+minimize disk I/O impact?); same AWS zone as client
+
+### Score Server
+
+TBD: spec system to hold scores over time
+
+### Datasets
+
+TBD: generated datasets should be park in S3 or somewhere for retrieval by URL
+
+## FAQ
+
+Question?
+
+Answer.
```

Comment:
I feel like this is a good opportunity to remove this empty section.

Reply:
Done